app/Platform/routes/platform-routes.php

```diff
@@ -1 +1,10 @@
 <?php
+
+use App\Platform\Http\Controllers\Auth\Social\GoogleAuthController;
+
+Route::group(['prefix' => '/platform'], function () {
+    Route::group(['prefix' => '/social'], function () {
```

Comment:
We won't need these prefixes, you don't need to group them, you can just use something like `/login/google` similar to how we do it for the other normal sign in endpoints.